src/main/webapp/assets/modules/create-event/createEventController.js

```diff
@@ -45,18 +45,41 @@ techlooper.controller("createEventController", function ($scope, $translate, jso
         }
         return false;
 
-      case "error-event-start-date":
+      case "error-past-date":
         if (!$scope.webinar) return false;
-        if ($scope.webinar.startDate && $scope.webinar.endDate) {
+        $scope.webinarForm.startDate.$setValidity("past", true);
+        $scope.webinarForm.endDate.$setValidity("past", true);
+
+        var isPast = false;
+
+        if ($scope.webinar.startDate) {
           var startDate = moment($scope.webinar.startDate, jsonValue.dateTimeFormat);
+          isPast |= startDate.isBefore(moment()) || startDate.isBefore(moment());
+          $scope.webinarForm.startDate.$setValidity("past", !isPast);
+        }
+
+        if ($scope.webinar.endDate) {
           var endDate = moment($scope.webinar.endDate, jsonValue.dateTimeFormat);
-          var before = endDate.isBefore(startDate);
-          $scope.webinarForm.startDate.$setValidity("range", !before);
-          $scope.webinarForm.endDate.$setValidity("range", !before);
-          return before;
+          isPast |= endDate.isBefore(moment()) || endDate.isBefore(moment());
+          $scope.webinarForm.endDate.$setValidity("past", !isPast);
         }
+
+        return isPast;
+
+      case "error-range-date":
+        if (!$scope.webinar) return false;
+        if ($scope.state("error-past-date")) return false;
         $scope.webinarForm.startDate.$setValidity("range", true);
         $scope.webinarForm.endDate.$setValidity("range", true);
+
+        if ($scope.webinar.startDate && $scope.webinar.endDate) {
+          var startDate = moment($scope.webinar.startDate, jsonValue.dateTimeFormat);
```

Comment:
Line is too long.<br>'startDate' is already defined.<br>'moment' is not defined.